integration/src/main/java/org/assimbly/integration/impl/CamelIntegration.java

```diff
@@ -654,12 +654,15 @@ public String addFlow(TreeMap<String, String> props)  {
 
 	private void addCustomActiveMQConnection(TreeMap<String, String> props, String frontendEngine) {
 		String activemqName = "activemq";
-		String activemqUrl = "tcp://localhost:61616";
+		String environment = props.get("environment");
+		String activemqUrl = String.format("tcp://assimbly-broker-%s:61616", environment);
```

Comment:
Ok. If environment is not specified it will use localhost.
In my local environment, the environment property is not set. There's another way to get this info?